openldap/defaults.yaml

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# vim: ft=yaml
+---
+openldap:
+  client_config:   /etc/ldap.conf
+  client_pkg:      openldap
+  server_defaults: /etc/default/slapd
+  server_pkg:      openldap
+  service:         slapd
+  database:        mdb
+  su_group:        root
+  index:           objectClass
```

Comment:
While this layout is aesthetically more pleasing, it doesn't work with the `yamllint` defaults rules that we're using across formulas:

* https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.colons

Basically, we're using the default of _one space_ after colons.  That would apply to all of the files in this PR's changeset.

Reply:
done